### ISSUE TITLE
Some bug extrator

### ISSUE THREAD

**Issue by ReyDonovan** [TrinityCore/TrinityCore#23748]
**Description:**
maybe this bug is mine only. I have a last client wow 820 and the classic wow installed in it, in this case I can not extract cards and db.2

Checked on a clean client wow 820 without the classic wow this error is not observed

CHANGEME Description of the problem or issue here.

**Current behaviour:**

CHANGEME Tell us what happens.
If this is a crash, post the crashlog (upload to https://gist.github.com/).

**Expected behaviour:**

CHANGEME Tell us what should happen instead.

**Steps to reproduce the problem:**

1. CHANGEME Step 1 include entries of affected creatures / items / quests with a link to the relevant wowhead page.  
2. Step 2
3. Step 3

**Branch(es):*master*

**TC rev. hash/commit:*https://github.com/TrinityCore/TrinityCore/commit/3df2e379eff98eba2c86bc6823cde1def6213afa* 

CHANGEME Copy the result of server debug command (if you need to run it from client get prat addon)

**Operating system:** CHANGEME OS

**Comment by MaxtorCoder** [comment]
Eh? If I recall correctly there was an argument which let's you select what product/installation to load
`wow` = Live
`wowt` = PTR
`wow_classic` = Classic

**Comment by ReyDonovan** [comment]
@MaxtorCoder 
![Новый точечный рисунок](https://user-images.githubusercontent.com/25749510/64004780-9cfca480-cb17-11e9-9170-5f4465987d2f.jpg)

**Comment by MaxtorCoder** [comment]
https://github.com/TrinityCore/TrinityCore/blob/master/src/tools/map_extractor/System.cpp#L156
Check this.

**Comment by ReyDonovan** [comment]
Yes, I don’t know how this happened, but when installing wow classic the old data was archived and new data was added, I think that somehow they work together in the same files 820 and 1.13

**Comment by MaxtorCoder** [comment]
Classic is basically 820, just different. Anyways try the argument I sent above, it should load the 820 part of the data folder.

**Comment by Shauren** [comment]
I wonder if its because your installation is not quite how a clean 8.2.0 install looks like (there should be no wow.exe in there)
Also what about build.info content

**Comment by ReyDonovan** [comment]
@Shauren im use official client https://pastebin.com/gDHThxpZ

**Comment by MaxtorCoder** [comment]
Ahh I see it. The Live version (820) is not in .build.info

**Comment by Shauren** [comment]
Yep, that is definitely the reason
